src/lando/api/tests/test_views.py

```diff
@@ -0,0 +1,47 @@
+from unittest import mock
+
+import pytest
+
+
+@pytest.mark.django_db(transaction=True)
+def test__views__git2hgCommitMapView(commit_maps, authenticated_client, monkeypatch):
+    mock_catch_up = mock.MagicMock()
+    monkeypatch.setattr("lando.api.views.CommitMap.catch_up", mock_catch_up)
+    response = authenticated_client.get(f"/api/git2hg/test_git_repo/{'a' * 40}")
```

Comment:
Tests should be using the unauthenticated client, since consumers in automation won't be authenticated.